src/cache.js

```diff
@@ -69,6 +69,25 @@ query getProjectsV2Fields($id: ID!, $cursor: String!) {
 	}
 }
 `;
+const GetRepositoryDescription = gql`
+query GetRepositoryDescription($orgName: String!, $repoName: String!){
+    repository(owner: $orgName, name: $repoName) {
+		description
+        main: object(expression: "main:README.md") {
+            ... on Blob {
+                text
+            }
+        }
+        master: object(expression: "master:README.md") {
+            ... on Blob {
+                text
+            }
+        }        
```

Comment:
You don't have to do this in 2 steps. Target `HEAD` rather than a branch and it gives you the repositories default branch

```suggestion
       readme: object(expression: "HEAD:README.md") {
            ... on Blob {
                text
            }
        }        
```